ecosystem/ffmpeg_plugin/0001-avdevice-kahawai-Rename-kahawai-indev-to-kahawai_dec.patch

```diff
@@ -0,0 +1,55 @@
+From bcddca9d7e7642a3e76f6d2b48bb7cd2a5fcc111 Mon Sep 17 00:00:00 2001
+From: "Yew, Chang Ching" <chang.ching.yew@intel.com>
+Date: Mon, 17 Apr 2023 10:49:27 +0800
+Subject: [PATCH 1/2] avdevice/kahawai: Add kahawai output device
```

Comment:
Can we merge the two patches into one?

Reply:
Patched merge into single patch https://github.com/OpenVisualCloud/Media-Transport-Library/pull/234/commits/dea527a90227f36a840ef826c3c71b0a0bcd86b6#diff-e1a04b2b79c9de9192b76420b9a0014cdc169ffc33b57e8219cec8d7ca2c1b77